pylint/extensions/_check_docs_utils.py

```diff
@@ -311,6 +311,14 @@ class GoogleDocstring(Docstring):
         [\(\[] [^\n]+ [\)\]]          # with the contents of the container
     """.format(type=re_type, xref=re_xref)
 
+    re_multiple_type = r"""
+        (?:{container_type}|{type})
+        (?:\s+or\s+(?:{container_type}|{type}))*
+    """.format(
+        type=re_type,
```

Comment:
Let's put it right after the upper parens, the indentation looks a bit weird as is.